scripts/main.js

```diff
@@ -0,0 +1,19 @@
+function saveItemToList() {
+    const getUserItemValue = document.getElementById('newItem').value;
+    if (getUserItemValue == 0) {
+        console.log('Nie podales wartosci')
+    } else {
```

Comment:
`else` is not needed in this case. If user did not provide any input, then you can use simple `return`: `if (blablabla) return;`

Whatever code will be after this line, it will not be executed when user does not provide any input.